plugins/vpp/model/l3/l3.proto

```diff
@@ -74,4 +74,22 @@ message STNTable {
         string interface = 2;                           /* interface where the packets will be punted */
     }
     repeated STNTableEntry stn_entries = 1;     /* list of STN entries */
+}
+
+/* Enables/disables IP neighbor scanning */
+message IPScanNeighbor {
+    enum Mode {                                         /* IP scan neighbor mode. Disabled by default, it can be enabled
+                                                           for IPv4 or IPv6 separately, or for both of them*/
+        DISABLED = 0;
+        IPv4 = 1;
+        IPv6 = 2;
+        BOTH = 3;
+    }
+    Mode mode = 1;
+
+    uint32 scan_interval = 2;
```

Comment:
Why choose types different from the ones in binary API? Takes extra space and needs conversion.